lib/tuner.lua

```diff
@@ -99,4 +90,28 @@ function tuner:redraw()
   end
 end
 
+function tuner:redraw()
+  local signal_strength = oscin.get_signal_strength()
+  -- noize_meter.set_ceiling(math.floor(util.linexp(0,1,1,40,signal_strength))) -- signal strength
+  noize_meter.set_ceiling(math.floor(util.explin(1,2,1,40,2-signal_strength))) -- noize strength
```

Comment:
toggle between the 2 above lines to switch between tracking signal strength or absence of it (i.e. noise + silence between stations)